anycloud/node/package.json

```diff
@@ -3,13 +3,16 @@
   "version": "0.1.41",
   "description": "CLI to elastically scale webservers in any cloud provider",
   "main": "dist/index.js",
+  "types": "index.d.ts",
   "bin": {
     "anycloud": "./bin/anycloud"
   },
   "scripts": {
     "build": "tsc",
-    "test": "echo \"Error: no test specified\" && exit 1",
-    "preinstall": "node ./get-anycloud.js"
+    "test": "jest",
+    "preinstall": "node ./get-anycloud.js",
+    "beforepublish": "jest && rm -rf bin && tsc",
```

Comment:
I don't think this one exists. You want [`prepublishOnly`](https://docs.npmjs.com/cli/v6/using-npm/scripts).

Reply:
Yes, it does not, just add it to run it manually temporally.